app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,15 @@
   <head>
     <title><%= content_for(:page_title) || "Samson" %></title>
     <%= stylesheet_link_tag    "application", media: "all" %>
+    <% Samson::Hooks.plugins.map(&:stylesheets).flatten.each do |css| %>
+      <%= stylesheet_link_tag asset_path(css) %>
+    <% end %>
+
     <%= javascript_include_tag "application" %>
+    <% Samson::Hooks.plugins.map(&:javascripts).flatten.each do |javascript| %>
```

Comment:
I haven't added any. I tried to ```erb``` [application.js](https://github.com/zendesk/samson/blob/fneves/buddy_request_plugin/app/assets/javascripts/application.js), since I believe that is should be Samson that will knowledge of which plugins to include. It didn't work since the assets will be precompiled before the erb translation takes effect, which would not include the javascripts from the plugin.
Is there a different way to add them to sprockets that would work?